custom-metrics-stackdriver-adapter/pkg/adapter/provider/provider.go

```diff
@@ -28,11 +28,11 @@ import (
 	"k8s.io/metrics/pkg/apis/custom_metrics"
 
 	// Install registers the API group and adds types to a scheme.
-	_ "k8s.io/client-go/pkg/api/install"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
Which scheme are you talking about here? The client-go clientset has its own scheme. Where do you need another scheme?

In general, you shouldn't depend on `k8s.io/kubernetes`.

Reply:
Removed kubernetes dependency